lib/VerticalDelegate.js

```diff
@@ -408,7 +408,8 @@ module.exports = {
 		if (!list.hasReset) return this.reset(list);
 
 		var cpp = this.controlsPerPage(list),
-			end = Math.max(list.$.page1.start, list.$.page2.start) + cpp;
+			end = Math.max(list.$.page1.start, list.$.page2.start) + cpp - 1,
+			idx = props.index - 1;
```

Comment:
But in this line, I have doubt for `-1`
`props.index` means starting index of added models.
If we had 38 models and then add 100 models, `props.index` is 38.